downstream.go

```diff
@@ -904,6 +910,7 @@ func (dc *downstreamConn) updateSupportedCaps() {
 			supportedCaps[cap] = supported && uc.caps[cap]
 		}
 	})
+	supportedCaps["draft/event-playback"] = dc.srv.LogPath != "" && dc.network != nil
```

Comment:
The `supportedCaps` variable is part of the `needAllDownstreamCaps` logic. I'd prefer to manually call `dc.setSupportedCap()` instead of abusing the `needAllDownstreamCaps` logic.